spec/poros/food_spec.rb

```diff
@@ -0,0 +1,162 @@
+                    "value": 1.82
+                },
+                {
+                    "nutrientId": 1005,
+                    "nutrientName": "Carbohydrate, by difference",
+                    "nutrientNumber": "205",
+                    "unitName": "G",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 20.0
+                },
+                {
+                    "nutrientId": 1008,
+                    "nutrientName": "Energy",
+                    "nutrientNumber": "208",
+                    "unitName": "KCAL",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 82.0
+                },
+                {
+                    "nutrientId": 2000,
+                    "nutrientName": "Sugars, total including NLEA",
+                    "nutrientNumber": "269",
+                    "unitName": "G",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 4.55
+                },
+                {
+                    "nutrientId": 1079,
+                    "nutrientName": "Fiber, total dietary",
+                    "nutrientNumber": "291",
+                    "unitName": "G",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 2.7
+                },
+                {
+                    "nutrientId": 1093,
+                    "nutrientName": "Sodium, Na",
+                    "nutrientNumber": "307",
+                    "unitName": "MG",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 55.0
+                },
+                {
+                    "nutrientId": 1257,
+                    "nutrientName": "Fatty acids, total trans",
+                    "nutrientNumber": "605",
+                    "unitName": "G",
+                    "derivationCode": "LCCS",
+                    "derivationDescription": "Calculated from value per serving size measure",
+                    "value": 0.0
+                }
+            ]
+        }
+      food = Food.new(attrs)
+      expect(food).to be_a Food
+      
+      expect(food.gtinupc.class).to eq(String)
+      expect(food.gtinupc).to eq("492111402857")
+
+      expect(food.description.class).to eq(String)
+      expect(food.description).to eq("SWEET POTATOES")
+
+      expect(food.brandowner.class).to eq(String)
+      expect(food.brandowner).to eq("ARCHER FARMS")
+
+      expect(food.ingredients.class).to eq(String)
+      expect(food.ingredients).to eq("SWEET POTATOES.")
```

Comment:
I don't think you really need to test the class of these attrs since you're checking what they're equal to anyway. But this is a super thorough test.. nice work.